data/korath/korath missions.txt

```diff
@@ -257,3 +257,179 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Stormhold Artifacts [1] Coded Message"
+	name "Go to <planet>"
+	description `Head to <planet> to deal with a "security issue."`
+	landing
+	to offer
+		has "event: stormhold korath issue"
+		not "Hai Reveal [B12] Devil-Hide Battle: accepted"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai" "Hai (Unfettered)" "Pirate"
+		not planet Stormhold
+	destination Stormhold
+	on offer
+		conversation
+			`As soon as you land, your communications console beeps urgently. It's a coded message from <planet> that simply reads, "Security issue. Return to <planet> at earliest convenience."`
+			`	The digital signature claims to be of diplomatic origin, and from the burning acorn insignia, it is most likely from the Unfettered.`
+				accept
+
+mission "Korath Stormhold Artifacts [2] Drones"
+	name "Destroy Korath drones at <waypoints>."
+	description `Destroy Korath stealth drones at <waypoints>. This is an optional mission; the Syndicate will do it if you don't.`
+	landing
+	source Stormhold
+	destination Stormhold
+	waypoint Almach
+	to offer
+		has "Korath Stormhold Artifacts [1] Coded Message: done"
+		not "Hai Reveal [B12] Devil-Hide Battle: accepted"
+	on offer
+		log "Factions" "Korath" `According to the Unfettered, the Korath were intelligent and well-cultured reptiles, until their civilization fell apart a few centuries ago. They peacefully coexisted with the Hai until they got desperate for resources and invaded. The Hai valiantly defended themselves against the Korath, but eventually the Korath nearly wiped themselves out through infighting. The few survivors live east of Syndicate space and frequently raid nearby human settlements.`
+		log `There was an abundance of Korath artifacts at Stormhold, but Batu fears Stormhold was merely a staging area and sales distribution site. He expects this is part of a larger Korath outfit sales operation that is based on a nearby planet.`
+		conversation
+			`Choot'k greets you, with a worried-looking Batu at his side. "Welcome back, Captain <last>! We've found many spoils of war on this planet. Sadly, there weren't any jump drives," he says, winking. "Fortunately, the pirates had plenty of other gadgetry littering their bunkers, including Hai and Korath outfits. We're gathering it to secure areas for inventory and analysis before divvying it up."`
```

Comment:
```suggestion
			`Choot'k greets you, with a worried-looking Batu at his side. "Welcome back, Captain <last>! We have found many spoils of war on this planet. Sadly, there weren't any jump drives," he says, winking. "Fortunately, the pirates had plenty of other gadgetry littering their bunkers, including Hai and Korath outfits. We are gathering it to secure areas for inventory and analysis before divvying it up."`
```
I dont think Hai should use contractions, especially not an unfettered

Reply:
Applied manually.